src/main/resources/org/jenkinsci/plugins/durabletask/executors/OnceRetentionStrategy/config.groovy

```diff
@@ -0,0 +1,7 @@
+package org.jenkinsci.plugins.durabletask.executors.OnceRetentionStrategy
+
+def f = namespace(lib.FormTagLib);
+
+f.entry(title: "Idle timeout", field: "idleMinutes") {
+    f.number(default: 0)
```

Comment:
@jglick i understood your point, i will think about removing durable-task support and not using this retention strategy in docker-plugin until jenkins and this plugin will provide normal API